src/Tizen.WebView/Tizen.WebView/PolicyDecisionEventArgs.cs

```diff
@@ -0,0 +1,189 @@
+﻿/*
+ * Copyright (c) 2019 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.ComponentModel;
+
+namespace Tizen.WebView
+{
+    /// <summary>
+    /// Enumeration values used to specify Policy Navigation Type.
+    /// </summary>
+    public enum PolicyNavigationType {
+        /// <summary>
+        /// Link clicked.
+        /// </summary>
+        LinkClicked = 0,
+        /// <summary>
+        /// Form submitted.
+        /// </summary>
+        FormSubmitted = 1,
+        /// <summary>
+        /// Back forward.
+        /// </summary>
+        BackForward = 2,
+        /// <summary>
+        /// Reload.
+        /// </summary>
+        Reload = 3,
+        /// <summary>
+        /// Form resubmitted.
+        /// </summary>
+        FormResubmitted = 4,
+        /// <summary>
+        /// Other.
+        /// </summary>
+        Other = 5
+    };
+
+    /// <summary>
+    /// Enumeration values used to specify Policy Decision Type.
+    /// </summary>
+    public enum PolicyDecisionType {
+        /// <summary>
+        /// Use.
+        /// </summary>
+        Use,
+        /// <summary>
+        /// Download.
+        /// </summary>
+        Download,
```

Comment:
What is means of Download?

Reply:
I can't understand. could you explain with some example?
What is means of `Ignore`?

Reply:
This enum used for all decision request event, but name is `NavigationPolicy`. 
Is it right?

Reply:
Ignore: If user (Application) does not want to continue loading Navigation Policy is set as Ignore, eg: for custom URLs like tizen://tizenstore/app.wgt
Use: If user (Application) wants to continue loading the webpage policy is set as use.
Download: when the URL is downloadable type ex: http://abc.com/filenamae.apk, Navigation Policy Set as Download and same is informed to Application, so that app will send the URL to downloader module

Reply:
no way to set NavigationPolicy